### PR TITLE
Fix for Video Table Example Building

### PR DESCRIPTION
**Pull request by ay8s**
Should fix up the Video Table Example where size is not found. Reported in Slack.

### CODE REVIEW

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please ensure license is correct for NicCageNode.mm:</p>
<pre><code>//
//  NicCageNode.mm
//  Texture
//
//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under the BSD-style license found in the
//  LICENSE file in the /ASDK-Licenses directory of this source tree. An additional
//  grant of patent rights can be found in the PATENTS file in the same directory.
//
//  Modifications to this file made after 4/13/2017 are: Copyright (c) 2017-present,
//  Pinterest, Inc.  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
</code></</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: examples_extra/VideoTableView/Sample/NicCageNode.mm [hunk 1]
```diff
@@ -167,8 +167,11 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
 {
   CGSize videoNodeSize = _isImageEnlarged ? CGSizeMake(2.0 * kImageSize, 2.0 * kImageSize)
                                           : CGSizeMake(kImageSize, kImageSize);
-  _videoNode.size = ASRelativeSizeRangeMakeWithExactCGSize(videoNodeSize);
-  _textNode.flexShrink = 1.0;
+  
+  [_videoNode.style setWidth:ASDimensionMake(videoNodeSize.width)];
+  [_videoNode.style setHeight:ASDimensionMake(videoNodeSize.height)];
```
Comment (maicki) [inline examples_extra/VideoTableView/Sample/NicCageNode.mm]: Can you please change that to: `_videoNode.style.preferredSize = videoNodeSize;`

**Comment by ay8s** [comment]
@maicki Adjusted that. 👍  

Used it previously but thought it wasn't working as the videos weren't all loading in. Adjusted the URLs for the two which no longer exist to use one from ASDKTube.

File: examples_extra/VideoTableView/Sample/NicCageNode.mm [hunk 2]
```diff
@@ -167,8 +167,11 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
 {
   CGSize videoNodeSize = _isImageEnlarged ? CGSizeMake(2.0 * kImageSize, 2.0 * kImageSize)
                                           : CGSizeMake(kImageSize, kImageSize);
-  _videoNode.size = ASRelativeSizeRangeMakeWithExactCGSize(videoNodeSize);
-  _textNode.flexShrink = 1.0;
+  
+  [_videoNode.style setHeight:ASDimensionMake(videoNodeSize.height)];
```
Comment (nguyenhuy) [inline examples_extra/VideoTableView/Sample/NicCageNode.mm]: There is no need to `setHeight` separately. Please remove this line.

File: examples_extra/VideoTableView/Sample/NicCageNode.mm [hunk 3]
```diff
@@ -96,13 +96,13 @@ - (instancetype)initWithKittenOfSize:(CGSize)size
...
       _videoNode = [[ASVideoNode alloc] init];
-      _videoNode.asset = [AVAsset assetWithURL:[NSURL URLWithString:@"https://files.parsetfss.com/8a8a3b0c-619e-4e4d-b1d5-1b5ba9bf2b42/tfss-753fe655-86bb-46da-89b7-aa59c60e49c0-niccage.mp4"]];
+      _videoNode.asset = [AVAsset assetWithURL:[NSURL URLWithString:@"https://www.w3schools.com/html/mov_bbb.mp4"]];
       break;
       
     case 1:
       // Construct the video node directly from the .mp4 URL
       _videoNode = [[ASVideoNode alloc] init];
-      _videoNode.asset = [AVAsset assetWithURL:[NSURL URLWithString:@"https://files.parsetfss.com/8a8a3b0c-619e-4e4d-b1d5-1b5ba9bf2b42/tfss-753fe655-86bb-46da-89b7-aa59c60e49c0-niccage.mp4"]];
+      _videoNode.asset = [AVAsset assetWithURL:[NSURL URLWithString:@"https://www.w3schools.com/html/mov_bbb.mp4"]];
```
Comment (nguyenhuy) [inline examples_extra/VideoTableView/Sample/NicCageNode.mm]: Please extract the URL to a constant.

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please ensure license is correct for NicCageNode.mm:</p>
<pre><code>//
//  NicCageNode.mm
//  Texture
//
//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under the BSD-style license found in the
//  LICENSE file in the /ASDK-Licenses directory of this source tree. An additional
//  grant of patent rights can be found in the PATENTS file in the same directory.
//
//  Modifications to this file made after 4/13/2017 are: Copyright (c) 2017-present,
//  Pinterest, Inc.  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
</code></</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by nguyenhuy** [comment]
Also, please fix Danger's warning(s). Thanks!

File: examples_extra/VideoTableView/Sample/NicCageNode.mm (hunk 2 above)
Comment (ay8s) [inline examples_extra/VideoTableView/Sample/NicCageNode.mm]: My bad, left that in. Needed some coffee ☕️

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please ensure license is correct for NicCageNode.mm:</p>
<pre><code>//
//  NicCageNode.mm
//  Texture
//
//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under the BSD-style license found in the
//  LICENSE file in the /ASDK-Licenses directory of this source tree. An additional
//  grant of patent rights can be found in the PATENTS file in the same directory.
//
//  Modifications to this file made after 4/13/2017 are: Copyright (c) 2017-present,
//  Pinterest, Inc.  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
</code></</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please ensure license is correct for NicCageNode.mm:</p>
<pre><code>//
//  NicCageNode.mm
//  Texture
//
//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under the BSD-style license found in the
//  LICENSE file in the /ASDK-Licenses directory of this source tree. An additional
//  grant of patent rights can be found in the PATENTS file in the same directory.
//
//  Modifications to this file made after 4/13/2017 are: Copyright (c) 2017-present,
//  Pinterest, Inc.  Licensed under the Apache License, Version 2.0 (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
</code></</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Pull request merged**